torchmetrics/utilities/warn.py

```diff
@@ -0,0 +1,18 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
Maybe move this to exceptions? having a new module just for one line... :rabbit: